grafeas-v1/README.md

```diff
@@ -0,0 +1,63 @@
+# Ruby Client for the Grafeas V1 API
+
+API Client library for the Grafeas V1 API
+
+The Grafeas API stores, and enables querying and retrieval of, critical metadata about all of your software artifacts.
+
+https://github.com/googleapis/google-cloud-ruby
+
+## Installation
+
+```
+$ gem install grafeas-v1
+```
+
+## Quick Start
+
+```ruby
+require "grafeas/v1"
+
+client = ::Grafeas::V1::Grafeas::Client.new
```

Comment:
Does this example still work without default credentials?

Reply:
It might, if the underlying implementation doesn't require credentials or has a different credentialing mechanism than Google's.